### PR TITLE
allow creation/fitting/testing of one state lineages

### PR DESCRIPTION
**Pull request by shak360**

### CODE REVIEW

**Comment by shak360** [comment]
Fixes #370

**Comment by shak360** [comment]
fixes #357

**Comment by shak360** [comment]
@aarmey Let me know if this is what you were expecting. The test file isn't being tested when pytest is run in the checks, but I was thinking it's just waiting to be pushed first or something.

File: lineage/CellVar.py [hunk 1]
```diff
@@ -158,7 +158,7 @@ def double(parent_state, T):
 
     # Rolling two of the same loaded dice separate times and assigning
     # where they landed to states
-    left_state_results, right_state_results = sp.multinomial.rvs(n=1, p=np.squeeze(T[parent_state, :]), size=2)
```
Comment (shak360) [inline lineage/CellVar.py:161]: Only thing that needed to be changed to allow for one state lineages.

File: lineage/tests/test_StateEstimation.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
+""" Unit test file. """
```
Comment (shak360) [inline lineage/tests/test_StateEstimation.py:1]: New unit test file with one state lineages/fitting and evaluation of parameter estimation.

**Comment by aarmey** [comment]
Looking into why the tests don't run...

**Comment by aarmey** [comment]
@shak360 OH! Methods that should run within the test class must start with `test_` in their name. Look at the other files to see what I mean.

**Comment by shak360** [comment]
Ah, got it. I'll make the change in a new branch.

**Pull request merged**